pwndbg/commands/msr.py

```diff
@@ -0,0 +1,181 @@
+from __future__ import annotations
+
+import argparse
+from typing import Optional
+from typing import Tuple
+
+import pwnlib.asm
+
+import pwndbg.aglib.file
+from pwndbg.commands import CommandCategory
+
+# Taken from linux/arch/x86/include/asm/msr-index.h
+# Arbitrary list, feel free to add more.
+X86_MSRS = {
+    "MSR_EFER": 0xC0000080,
+    "MSR_STAR": 0xC0000081,
+    "MSR_LSTAR": 0xC0000082,
+    "MSR_CSTAR": 0xC0000083,
+    "MSR_SYSCALL_MASK": 0xC0000084,
+    "MSR_FS_BASE": 0xC0000100,
+    "MSR_GS_BASE": 0xC0000101,
+    "MSR_KERNEL_GS_BASE": 0xC0000102,
+    "MSR_TSC_AUX": 0xC0000103,
+}
+
+
+COMMON_MSRS = {"x86": X86_MSRS, "x86-64": X86_MSRS}
+
+
+def parse_msr(msr: str, arch: str) -> Optional[int]:
+    # first try to parse as int, then as hex, then as MSR name
+    try:
+        msr_val = int(msr)
+    except ValueError:
+        try:
+            msr_val = int(msr, 16)
+        except ValueError:
+            if msr not in COMMON_MSRS[arch]:
+                print(f"unknown MSR {msr} on {arch}")
+                return None
+
+            msr_val = COMMON_MSRS[arch][msr]
+
+    return msr_val
```

Comment:
```suggestion
def parse_msr(msr: str, arch: str) -> Optional[int]:
    # first try to parse MSR name, then as int/hex
    if (val := COMMON_MSRS.get(arch, {}).get(msr)) is not None:
        return val
    
    try:
        return int(msr, 0)
    except ValueError:
        print(f"unknown MSR {msr} on {arch}")
        return None
```
@disconnect3d do you like this suggestion 🤔?